Cant build AWX development environment

### Please confirm the following

- [X] I agree to follow this project's [code of conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html).
- [X] I have checked the [current issues](https://github.com/ansible/awx/issues) for duplicates.
- [X] I understand that AWX is open source software provided for free and that I might not receive a timely response.

### Bug Summary

For some reason, that I dont understand, loads of ISP's in India have blocked `raw[.]githubusercontent[.]com`. More info here (sorry for the news link below):

https://timesofindia.indiatimes.com/gadgets-news/github-content-domain-blocked-for-these-indian-users-reports/articleshow/96687992.cms

But because of this blocking I can't build awx locally because we have two conf files that we read from githubusercontent:

```
DOCKER_BUILDKIT=1 docker build -t ghcr.io/ansible/awx_devel:bulk_api \
    --build-arg BUILDKIT_INLINE_CACHE=1 \
    --cache-from=ghcr.io/ansible/awx_devel:bulk_api .
[+] Building 1.3s (9/47)                                                                                                                                                                                                                      
 => [internal] load build definition from Dockerfile                                                                                                                                                                                     0.0s
 => => transferring dockerfile: 97B                                                                                                                                                                                                      0.0s
 => [internal] load .dockerignore                                                                                                                                                                                                        0.0s
 => => transferring context: 93B                                                                                                                                                                                                         0.0s
 => [internal] load metadata for quay.io/centos/centos:stream9                                                                                                                                                                           1.2s
 => importing cache manifest from ghcr.io/ansible/awx_devel:bulk_api                                                                                                                                                                     0.0s
 => ERROR https://raw.githubusercontent.com/containers/libpod/master/contrib/podmanimage/stable/containers.conf                                                                                                                          0.0s
 => ERROR https://raw.githubusercontent.com/containers/libpod/master/contrib/podmanimage/stable/podman-containers.conf                                                                                                                   0.0s
 => [internal] load build context                                                                                                                                                                                                        0.0s
 => => transferring context: 2.74kB                                                                                                                                                                                                      0.0s
 => FROM quay.io/ansible/receptor:devel                                                                                                                                                                                                  0.0s
 => CANCELED [builder 1/9] FROM quay.io/centos/centos:stream9@sha256:1f6b2e46a49fe61aac5d60f2ada8d0fcd4c92416a17c20123e1666887fd7b37d                                                                                                    0.0s
 => => resolve quay.io/centos/centos:stream9@sha256:1f6b2e46a49fe61aac5d60f2ada8d0fcd4c92416a17c20123e1666887fd7b37d                                                                                                                     0.0s
 => => sha256:7618dcae547fae7d6ceb78114d9fbbcf47d5a953282054ce9eedd5d845aafa76 350B / 350B                                                                                                                                               0.0s
 => => sha256:6f41e24aa37b012f53e0b887169fceca11834d4f3c542fd5a6f361a433b78dc7 1.16kB / 1.16kB                                                                                                                                           0.0s
 => => sha256:1f6b2e46a49fe61aac5d60f2ada8d0fcd4c92416a17c20123e1666887fd7b37d 858B / 858B                                                                                                                                               0.0s
------
 > https://raw.githubusercontent.com/containers/libpod/master/contrib/podmanimage/stable/containers.conf:
------
------
 > https://raw.githubusercontent.com/containers/libpod/master/contrib/podmanimage/stable/podman-containers.conf:
------
failed to load cache key: Get "https://raw.githubusercontent.com/containers/libpod/master/contrib/podmanimage/stable/podman-containers.conf": read tcp [2405:201:1016:6023:b22:6744:88e9:16e5]:41498->[2606:50c0:8000::154]:443: read: connection reset by peer
make: *** [Makefile:533: docker-compose-build] Error 1
```

Is there any alternative we can use instead of `raw[.]githubusercontent[.]com`. This will affect loads of users in India I guess ?

### AWX version

latest

### Select the relevant components

- [ ] UI
- [ ] API
- [ ] Docs
- [ ] Collection
- [ ] CLI
- [X] Other

### Installation method

docker development environment

### Modifications

no

### Ansible version

_No response_

### Operating system

_No response_

### Web browser

_No response_

### Steps to reproduce

If you are in India, run the `make docker-compose-build`. 

### Expected results

The build should be created

### Actual results

I get an error:

```
DOCKER_BUILDKIT=1 docker build -t ghcr.io/ansible/awx_devel:bulk_api \
    --build-arg BUILDKIT_INLINE_CACHE=1 \
    --cache-from=ghcr.io/ansible/awx_devel:bulk_api .
[+] Building 1.3s (9/47)                                                                                                                                                                                                                      
 => [internal] load build definition from Dockerfile                                                                                                                                                                                     0.0s
 => => transferring dockerfile: 97B                                                                                                                                                                                                      0.0s
 => [internal] load .dockerignore                                                                                                                                                                                                        0.0s
 => => transferring context: 93B                                                                                                                                                                                                         0.0s
 => [internal] load metadata for quay.io/centos/centos:stream9                                                                                                                                                                           1.2s
 => importing cache manifest from ghcr.io/ansible/awx_devel:bulk_api                                                                                                                                                                     0.0s
 => ERROR https://raw.githubusercontent.com/containers/libpod/master/contrib/podmanimage/stable/containers.conf                                                                                                                          0.0s
 => ERROR https://raw.githubusercontent.com/containers/libpod/master/contrib/podmanimage/stable/podman-containers.conf                                                                                                                   0.0s
 => [internal] load build context                                                                                                                                                                                                        0.0s
 => => transferring context: 2.74kB                                                                                                                                                                                                      0.0s
 => FROM quay.io/ansible/receptor:devel                                                                                                                                                                                                  0.0s
 => CANCELED [builder 1/9] FROM quay.io/centos/centos:stream9@sha256:1f6b2e46a49fe61aac5d60f2ada8d0fcd4c92416a17c20123e1666887fd7b37d                                                                                                    0.0s
 => => resolve quay.io/centos/centos:stream9@sha256:1f6b2e46a49fe61aac5d60f2ada8d0fcd4c92416a17c20123e1666887fd7b37d                                                                                                                     0.0s
 => => sha256:7618dcae547fae7d6ceb78114d9fbbcf47d5a953282054ce9eedd5d845aafa76 350B / 350B                                                                                                                                               0.0s
 => => sha256:6f41e24aa37b012f53e0b887169fceca11834d4f3c542fd5a6f361a433b78dc7 1.16kB / 1.16kB                                                                                                                                           0.0s
 => => sha256:1f6b2e46a49fe61aac5d60f2ada8d0fcd4c92416a17c20123e1666887fd7b37d 858B / 858B                                                                                                                                               0.0s
------
 > https://raw.githubusercontent.com/containers/libpod/master/contrib/podmanimage/stable/containers.conf:
------
------
 > https://raw.githubusercontent.com/containers/libpod/master/contrib/podmanimage/stable/podman-containers.conf:
------
failed to load cache key: Get "https://raw.githubusercontent.com/containers/libpod/master/contrib/podmanimage/stable/podman-containers.conf": read tcp [2405:201:1016:6023:b22:6744:88e9:16e5]:41498->[2606:50c0:8000::154]:443: read: connection reset by peer
make: *** [Makefile:533: docker-compose-build] Error 1
```

### Additional information

_No response_

--- a/tools/ansible/roles/dockerfile/templates/Dockerfile.j2
+++ b/tools/ansible/roles/dockerfile/templates/Dockerfile.j2
@@ -208,8 +208,8 @@ ADD tools/docker-compose/start_tests.sh /start_tests.sh
 ADD tools/docker-compose/bootstrap_development.sh /usr/bin/bootstrap_development.sh
 ADD tools/docker-compose/entrypoint.sh /entrypoint.sh
 ADD tools/scripts/config-watcher /usr/bin/config-watcher
-ADD https://raw.githubusercontent.com/containers/libpod/master/contrib/podmanimage/stable/containers.conf /etc/containers/containers.conf
-ADD https://raw.githubusercontent.com/containers/libpod/master/contrib/podmanimage/stable/podman-containers.conf /var/lib/awx/.config/containers/containers.conf
+ADD tools/docker-compose/containers.conf /etc/containers/containers.conf
+ADD tools/docker-compose/podman-containers.conf /var/lib/awx/.config/containers/containers.conf
 {% else %}
 ADD tools/ansible/roles/dockerfile/files/launch_awx.sh /usr/bin/launch_awx.sh
 ADD tools/ansible/roles/dockerfile/files/launch_awx_task.sh /usr/bin/launch_awx_task.sh

--- a/tools/docker-compose/containers.conf
+++ b/tools/docker-compose/containers.conf
@@ -1,0 +1,12 @@
+[containers]
+netns="host"
+userns="host"
+ipcns="host"
+utsns="host"
+cgroupns="host"
+cgroups="disabled"
+log_driver = "k8s-file"
+[engine]
+cgroup_manager = "cgroupfs"
+events_logger="file"
+runtime="crun"

--- a/tools/docker-compose/podman-containers.conf
+++ b/tools/docker-compose/podman-containers.conf
@@ -1,0 +1,5 @@
+[containers]
+volumes = [
+	"/proc:/proc",
+]
+default_sysctls = []

